src/oauthHandlers/tokenHandlerStrategyClasses/tokenHandlerClient.js

```
@@ -159,7 +159,7 @@ class TokenHandlerClient {
           );
           for (let key in decodedLaunch) {
             if (!responseBody[key]) {
-              responseBody[key] = decodedLaunch[key];
+              responseBody[key] = String(decodedLaunch[key]);
```

Modifying this section (and actually line 154 above which also would require similar casting) only resolves the /token response payload, however subsequent fetches during token validation would still result in possible integer launch context.

Instead, we want to shift this validation left prior to the saving of the launch context in the first place.

https://github.com/department-of-veterans-affairs/lighthouse-oauth-proxy/blob/master/src/oauthHandlers/tokenHandlerStrategyClasses/saveDocumentStrategies/saveDocumentLaunchStrategy.js

This does add a bit of complexity since right now we aren't doing any validation on the provided launch context.

We should check that it is either 

If Base64 encoded json
- If there a key of `patient`, and in that case validate to a string is provided

If Non-base64 encoded json
- Assumed to be the patient, validate that the provided value is a string


If either scenario finds an integer, it should return a 400 error like below.

```
{
    "error": "invalid_launch",
    "error_description": "The provided patient launch must be a string."
}
```